h2o-genmodel/src/main/java/hex/genmodel/ModelMojoReader.java

```diff
@@ -157,6 +161,7 @@ private void readAll() throws IOException {
     _model._mojo_version = ((Number) readkv("mojo_version")).doubleValue();
     checkMaxSupportedMojoVersion();
     readModelData();
+    extractModelDetails();
```

Comment:
@Pscheidl please only read model details if they actually needed - in vast majority of use cases you only use MOJO for scoring, this creates additional overhead for mojo load and the structures will take additional memory - serializing the java objects will also be slower